lib/management/applyWorkflowUpdates.js

```diff
@@ -1,5 +1,23 @@
 const async = require('async');
 
+function getNewBucketWorkflows(serviceName, workflows, bucketName) {
+    const newBucketWorkflows =
+        (workflows &&
+        workflows[serviceName] &&
+        workflows[serviceName][bucketName]) || [];
```

Comment:
I find this construct confusing but very handy. Is it a widely used js idiom?

Reply:
I think it's widely used indeed, as an object or array are always truish in JS, and the `&&` operator is lazy it will stop at the first falsish value (or the last one) and return it. The `||` is the opposite, stop at the first truish value (or the last one) and return it.

Reply:
❤️JS

Reply:
Got it. Not only lazy, what was surprising to my eyes was the train of conjunctions and its variability in the returned type